pkg/gatewayserver/gatewayserver.go

```diff
@@ -206,6 +214,24 @@ func New(c *component.Component, conf *Config, opts ...Option) (gs *GatewayServe
 
 	hooks.RegisterUnaryHook("/ttn.lorawan.v3.NsGs", cluster.HookName, c.ClusterAuthUnaryHook())
 
+	for name, prefix := range gs.forward {
+		str := strings.Split(name, ":")
+		if len(str) != 2 {
+			continue
```

Comment:
Can't break config within v3 major, so we need to handle legacy config here (a `name` of `""`)